csrc/multidevice/device_mesh.h

```diff
@@ -40,6 +40,10 @@ class DeviceMesh final {
     return std::find(vector_.begin(), vector_.end(), device) != vector_.end();
   }
 
+  bool operator==(const DeviceMesh& other) const {
```

Comment:
Is it necessary to explicitly define this?

Reply:
not necessary, it was for convenience, but let me remove it